kernel/src/Boot/Patches/Patches405.cpp

```diff
@@ -193,9 +190,5 @@ void Mira::Boot::Patches::install_prerunPatches_405()
 	kmem[2] = 0x00;
 	kmem[3] = 0x00;
 
-	// prtinf hook patches
-	kmem = (uint8_t *)&gKernelBase[0x0034766E];
```

Comment:
Why was this removed? This is to enable verbose vsnprintf call correct?

Reply:
`// Enable *all* debugging logs (in vprintf)` Chops the portion of the function that these patches effect out so they no longer (or rather never) did anything